[Task]: Deprecate HeadlessChrome processor (#14572)
pimcore/pimcore: doc/Development_Documentation/23_Installation_and_Upgrade/07_Updating_Pimcore/11_Preparing_for_V11.md

          target: 'symfony-config'
          options:
            directory: '/var/www/html/var/config/custom_reports'
    ```
  

> - [Web2Print] Please keep in mind that deprecated `HeadlessChrome` will need to be replaced with the new processor `Chrome` when starting using 11.

Reviewer: ```suggestion
- [Web2Print] Please keep in mind that the deprecated processor `HeadlessChrome` needs to be replaced with the new processor `Chrome` in Pimcore 11.
```
